### ISSUE TITLE
parallel: Error: Command (--results) starts with '-'. Is this a wrong option?

### ISSUE THREAD

**Issue by HuJane** [jts/nanopolish#385]
Dear Jts,
I have successfully installed nanopolish and parallel (the latest versions), but when I run the parallel command, the following error was appeared.
![image](https://user-images.githubusercontent.com/17492179/40105027-67720f5a-5924-11e8-9675-42d12880d065.png)
When I ran the commands, parallel and nanopolish_makerange.py, separately —— it looks fine. I cannot figure out why they cannot be combined by '|'? Could you pls help me? Thank you!
![image](https://user-images.githubusercontent.com/17492179/40106179-a427b5e6-5927-11e8-926c-0c93fb283587.png)

**Comment by jts** [comment]
Please post the full command you tried to run, I can't see the whole thing.

Jared

**Comment by HuJane** [comment]
Hi Jared,
My full command:
python /home/hujie/software/nanopolish/scripts/nanopolish_makerange.py canu/Normal.contigs.fasta | parallel --results nanopolish.results -P 6 nanopolish variants --consensus polished_Normal.{1}.fa -w {1} -r Normal.fasta -b Normal.sorted.bam -g canu/Normal.contigs.fasta -t 8 --min-candidate-frequency 0.1 --methylation-aware dcm,dam

**Comment by HuJane** [comment]
$ python /home/hujie/software/nanopolish/scripts/nanopolish_makerange.py canu/Normal.contigs.fasta | parallel --results nanopolish.results -P 6 nanopolish variants --consensus polished_Normal.{1}.fa -w {1} -r Normal.fasta -b Normal.sorted.bam -g canu/Normal.contigs.fasta -t 8 --min-candidate-frequency 0.1 --methylation-aware dcm,dam
parallel: Error: Command (--results) starts with '-'. Is this a wrong option?

If commands ran separately:
python /home/hujie/software/nanopolish/scripts/nanopolish_makerange.py canu/Normal.contigs.fasta
tig00000012:0-891
tig00000025:0-388
tig00000047:0-662
tig00000048:0-727
tig00000052:0-1165
tig00000065:0-714
tig00000066:0-897
tig00000077:0-1213
tig00000171:0-615
tig00000253:0-1863
tig00000254:0-1176
tig00000255:0-545
tig00000256:0-1530
tig00000832:0-412
tig00000833:0-1148
tig00000834:0-686
tig00000835:0-1546
tig00000836:0-1027
tig00000837:0-1550
tig00000838:0-703
tig00000839:0-1206
tig00000840:0-836
tig00000841:0-446
tig00000842:0-366
tig00000843:0-475
tig00000844:0-479
tig00000845:0-1098
tig00000846:0-2790

$ parallel --results nanopolish.results -P 6 nanopolish variants --consensus polished_Normal.{1}.fa -w {1} -r Normal.fasta -b Normal.sorted.bam -g canu/Normal.contigs.fasta -t 8 --min-candidate-frequency 0.1 --methylation-aware dcm,dam
parallel: Input is read from the terminal. Only experts do this on purpose. Press CTRL-D to exit.

**Comment by jts** [comment]
It looks like you are using a different version of `parallel` than we expect. You should be using GNU parallel. See this discussion here:

https://stackoverflow.com/questions/16448887/gnu-parallel-not-working-at-all

**Comment by ole-tange** [comment]
Your version of GNU Parallel is too old. `--results` was introduced in 20121022.

**Comment by HuJane** [comment]
Yes, I used GNU parallel old version. After upgrade to 20150322, the problem is solved.

https://github.com/martinda/gnu-parallel
wget http://ftpmirror.gnu.org/parallel/parallel-20150322.tar.bz2
bzip2 -dc parallel-20150322.tar.bz2 | tar xvf -
cd parallel-20150322
./configure --prefix=${PWD} && make && make install
